src/Cppyy.h

```diff
@@ -324,7 +324,7 @@ namespace Cppyy {
     CPPYY_IMPORT
     std::string GetDatamemberTypeAsString(TCppScope_t var);
     CPPYY_IMPORT
-    intptr_t    GetDatamemberOffset(TCppScope_t var);
+    intptr_t    GetDatamemberOffset(TCppScope_t var, TCppScope_t parent = nullptr);
```

Comment:
We can get the enclosing scope from var here, can't we?

Reply:
It is not possible in the case of derived class.
`var->getParent()` will return the base class.
But we want to calculate the offset with respect to the derived class.

Reply:
In that case then we should change the name of the parameter. It is not `parent` according to your description.